src/SupportMessage/SupportMessage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+import { lighten } from 'polished'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { theme } from '../theme'
+
+export type SupportMessageProps = {
+  className?: string
+  children: string
```

Comment:
Instead of using `children`, should we use a named prop? Like `description`? It would make it more explicit

Reply:
Agree, we should call this `description`